examples/macos/text_edit_item.py

```diff
@@ -31,14 +31,23 @@
 
 """Run some automations to test things"""
 from __future__ import unicode_literals
-from __future__ import print_function
-import sys
-sys.path.append('/Users/daria/pywinauto')
-
-from pywinauto import application
-from pywinauto import macos 
+from __future__ import print_function
 
-app = application.Application()
-app.start('TextEdit')
-app.kill()
+import sys
 
+sys.path.append(".")
+
+from pywinauto.macos.application import Application
+
+app = Application()
+app.start('TextEdit')
+text_edit = app.Window_().TextEdit

```

Comment:
`Window_` is deprecated on Windows and should be removed in 0.7.0. Why it does exist in macOS code?